elastic/kibana
# [IM] Add missing values to the index details flyout

Fixes https://github.com/elastic/kibana/issues/123714

## Summary

This PR adds back missing values for "Docs Deleted" and "Primary Storage Size" displayed in the index details flyout. These values have probably been removed in this [PR#28855](https://github.com/elastic/kibana/pull/28855) but can found in [Index Stats API](https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-stats.html). 

## How to test
1. Start ES and Kibana with `yarn es snapshot` and `yarn start` 
2. Add some sample data 
3. Navigate to Index Management and check index details flyout
4. To delete a document, first use `GET /<index_name>/_search` in Console and copy `_id` of any document in the `hits` array, then use `DELETE /kibana_sample_data_ecommerce/_doc/<doc_id>?refresh=true` to delete a document
5. Check that the value in the flyout is updated (NOTE: For some reason, the count for deleted docs is incremented by 2 for each deleted document. It has something to do with index refresh and changes if the index is flushed)

## Screenshot

<img width="798" alt="Screenshot 2022-02-04 at 18 00 14" src="https://user-images.githubusercontent.com/6585477/152576661-cd609e71-912b-4c4c-9b45-0f325cd70447.png">

### Checklist

## Release Note
In Index Management app, index details again display previously missing values for the number of deleted documents and the primary storage size.
